Compilation failure on Aarch64 Fedora Linux because of missing `#include <cstdint>`

https://download.copr.fedorainfracloud.org/results/jdanek/skupper-router/fedora-rawhide-aarch64/03820885-skupper-router/builder-live.log.gz

```
/builddir/build/BUILD/skupper-router/tests/cpp-stub/cpp_stub.h:47:11: error: 'uint32_t' was not declared in this scope
   47 |         ((uint32_t*)fn)[0] = 0x58000040 | 9;\
      |           ^~~~~~~~
/builddir/build/BUILD/skupper-router/tests/cpp-stub/cpp_stub.h:248:13: note: in expansion of macro 'REPLACE_FAR'
  248 |             REPLACE_FAR(this, fn, fn_stub);
      |             ^~~~~~~~~~~
/builddir/build/BUILD/skupper-router/tests/cpp-stub/cpp_stub.h:26:1: note: 'uint32_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
   25 | #include <map>
  +++ |+#include <cstdint>
```

The above is my copy of the files. The corresponding lines in this project are at

https://github.com/coolxv/cpp-stub/blob/691b6251575e0af4df7f91aef932cbcff257ce84/src/stub.h#L40-L41

--- a/src/stub.h
+++ b/src/stub.h
@@ -12,6 +12,7 @@
 #endif
 //c
 #include <cstddef>
+#include <cstdint>
 #include <cstring>
 //c++
 #include <map>

